### ISSUE TITLE
Gantt chart for tasks: add more info (tasks dependencies and subtasks showing)

### ISSUE THREAD

**Issue by mykytavoronkov** [kanboard/kanboard#1187]
Please add more info for Gantt chart for tasks: add more info: 
- tasks dependencies based on links already implemented between tasks

Finish to Start (FS) - the default: The task cannot start before its predecessor ends, although it may start later. This is the most common type of relationship, and is described above.

Start to Start (SS): The task cannot start until the predecessor starts, although it may start later. This can be useful if you have a task whose start date depends on the start date of another task.

Finish to Finish (FF): The task cannot end before the predecessor ends, although it may end later.

Start to Finish (SF): The task cannot end before the predecessor starts, although it may end later. This task relationship is rarely used.
- subtasks showing grouped by task http://www.gantt.com/creating-gantt-charts.htm

No dependensies between subtasks! Only between tasks.

Thanks a lot for Gantt chart! :)

**Comment by Joan-Andreu** [comment]
This one is for us, the killing task ! 

It's the feature we need to be able to replace our "gantter for google drive".

**Comment by mkauzlar** [comment]
+1

**Comment by qualiacode** [comment]
+1

**Comment by derWalter** [comment]
+1

**Comment by fguillot** [comment]
Please, move your ticket to the Gantt plugin bug tracker: https://github.com/kanboard/plugin-gantt/issues

- This will facilitate the work of the maintainers (bug triage)
- Make sure your ticket still relevant with the latest version of the software
- If you are asking another feature request, do not hesitate to contribute, Kanboard is an open source project. Remember the famous quote from Linus Torvalds: _“Talk is cheap. Show me the code.”_

Thank you for your understanding
